src/components/map/MapService.js

```diff
@@ -47,7 +47,7 @@ goog.require('ga_urlutils_service');
           }
           if (minResolution) { // we remove useless resolutions
             for (var i = 0, ii = resolutions.length; i < ii; i++) {
-              if (resolutions[i] === i) {
+              if (resolutions[i] === minResolution) {
```

Comment:
The bug is the function using minResolution does nothingl but I haven't found the effect of this. May be just a misleading property.